.github/workflows/release-cli.yml

```diff
@@ -86,9 +93,151 @@ jobs:
           name: artifacts-${{ matrix.GOOS }}-${{ matrix.GOARCH }}
           path: bin/*
 
+  package-deb:
+    name: Build deb package
+    strategy:
+      matrix:
+        include:
+          - GOOS: linux
+            GOARCH: amd64
+            pkgArch: amd64
+          - GOOS: linux
+            GOARCH: arm64
+            pkgArch: arm64
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Get tag name
+        uses: olegtarasov/get-tag@2.1.3
+        id: tagName
+        with:
+          tagRegex: "cli/v(.*)"
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 22
+      - uses: pnpm/action-setup@v3
+        with:
+          version: 9
+      - uses: actions/setup-go@v5
+        with:
+          go-version: "1.23.4"
+          cache: false
+      - name: Run build script
+        run: make build VERSION=${{ steps.tagName.outputs.tag }}
+
+      - name: create dir structure for .deb package
+        run: |
+          mkdir -p .debpkg/usr/bin
```

Comment:
Just wondering - why did you use a hidden directory here?

Reply:
the example contained it, so just used that. we could use any folder name we want. 
and yes "/usr/bin" is the location where the binary get's installed.